.changeset/lovely-moments-join.md

```diff
@@ -1,12 +1,12 @@
 ---
-"@biomejs/biome": patch
+"@biomejs/biome": minor
```

Comment:
Can you revert this? Nursery rule can go out in patches https://biomejs.dev/linter/#nursery